models/ed/R/model2netcdf.ED2.R

```diff
@@ -280,9 +280,19 @@ read_T_files <- function(yr, yfiles, tfiles, outdir, start_date, end_date, ...){
     }
   }
   
-  CheckED2Version <- function(nc) {
+  CheckED2Variables <- function(nc) {
+    vars_detected <- NULL
+  
     if ("FMEAN_BDEAD_PY" %in% names(nc$var)) {
-      return("Git")
+      vars_detected <- c(vars_detected,"FMEAN_BDEAD_PY")
+      return("Contains_FMEAN")
+    }
+    if ("FMEAN_SOIL_TEMP_PY" %in% names(nc$var)) {
+      vars_detected <- c(vars_detected, "FMEAN_SOIL_TEMP_PY")
+      return("Contains_FMEAN")
+    }
+    if(!is.null(vars_detected)){
```

Comment:
`vars_detected` is set `NULL` in the beginning and the only two conditions where it becomes not NULL returns from the function already, how this `if` will ever be evaluated? What was the intended behavior?

Reply:
True! I'll fix it.

Reply:
Thanks Istem